bin/merge_script

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+#TODO:
+#List up the PRs these changes are from (so it is easy to see what to include in the cv32e40s/dev to cv32e40x/dev merge)
+
+#Variables:
+date_time=$(date +%Y.%m.%d-%k.%M)
+
+
+usage() {
+
+  echo "usage: $0 --[s_into_x-dv|x-dv_into_s|sdev_into_xdev|xdev_into_sdev]"
+  echo "--s_into_x-dv     Do a merge of core-v-verif cv32e40s/dev cv32e40s directory into cv32e40x-dv main (make sure the clonetb script has run)"
+  echo "--x-dv_into_s     Do a merge of cv32e40x-dv main into core-v-verif cv32e40s/dev cv32e40s (not yet developed)"
+  echo "--sdev_into_xdev  Do a merge of core-v-verif cv32e40s/dev into core-v-verif cv32e40x/dev"
+  echo "--xdev_into_sdev  Do a merge of core-v-verif cv32e40x/dev into core-v-verif cv32e40s/dev"
+
+  exit 1
+}
+
+#Merge of cv32e40s into cv32e40x-dv:
+merge_cv32e40s_into_cv32e40x-dv () {
+
+  echo "Merge of cv32e40s into cv32e40x-dv:"
+
+  #Enter the cv32e40x-dv repo in cv32e40x subdirectory
+  cd cv32e40x
+
+  #Make a branch in cv32e40x-dv that contain core-v-verif's cv32e40s folder from the cv32e40s/dev branch
```

Comment:
Maybe `#` could be substituted with `echo `, in most places?

(Or maybe `echo "==== $0:"` or something even more easily identifiable.)

Reply:
Good idea